frontend/package.json

```diff
@@ -48,5 +46,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/jest": "^29.4.0",
+    "jest": "^27.5.1",
```

Comment:
ts-jest seems very upset by this version of jest. `"jest": "^29.4.2",` makes it happy.